sass/blocks/intro/_intro.scss

```diff
@@ -0,0 +1,23 @@
+.intro {
+  height: 39rem;
+  overflow: hidden;
+  background: url("../images/intro-background.png") center;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  &-images {
+    height: 92%;
+    background: url("../images/intro-sedona-small.png") center no-repeat;
```

Comment:
More or less the technic you used here is fine, and I think it's good. You have just one image with three text lines. Maybe just 92% looks a bit disturbing and not nice. I understand this magical number you used to put your image in the centre to look like on designs dimensionally, instead, you probably should have used background properties, you can set size and position of the background image, you could have even set it as the second image on .intro block I bet. I think it should be improved. But I need to tell you that you'll get this texts as separate three .svg images, so you I would recommend having it as three separate blocks like I think we had before when we're creating the layout. I will live a comment in html, then you might also explore another image replacement technique as this one would require you to have separate tags for text and images, unless you use multiple backgrounds if it's possible, not sure but I will leave it here https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds